server/config/config.go

```diff
@@ -129,6 +129,8 @@ type Config struct {
 
 	logger   *zap.Logger
 	logProps *log.ZapProperties
+
+	EnableConfigManager bool
```

Comment:
I think we can enable it by default. No need the config. If no client use it, the feature has no side effect.

Reply:
The switch will also control the PD config management which will start a goroutine in #2049.